dev

```diff
@@ -177,36 +179,51 @@ def kind_create(recreate):
         error_callback=_log_wait_node_timeout,
     )
 
-    print("Installing Helm's tiller asynchronously in the cluster.")
-    _run(
-        cmd=[
-            "kubectl", "create", "serviceaccount", "tiller",
-            "--namespace", "kube-system",
-        ],
-        print_end="",
-    )
-    _run(
-        cmd=[
-            "kubectl", "create", "clusterrolebinding", "tiller",
-            "--clusterrole", "cluster-admin",
-            "--serviceaccount", "kube-system:tiller",
-        ],
-        print_end="",
-    )
-    _run([
-        "helm", "init",
-        "--service-account", "tiller",
-    ])
+    if os.getenv('Z2JH_HELM_VERSION', '').startswith('3.'):
+        print("Creating K8s namespace for Helm 3 to work in.")
+        k8s_ns = {
+            "apiVersion": "v1",
+            "kind": "Namespace",
+            "metadata": { "name": os.environ["Z2JH_KUBE_NAMESPACE"] },
+        }
+        _run(
+            cmd=[
+                "sh", "-c", "echo '{}' | kubectl create -f -".format(
+                    json.dumps(k8s_ns))
+            ],
+            print_end="",
+        )
```

Comment:
Perhaps `kubectl create namespace NAME` is more suitable here? I'm impressed you got the current solution working!

Reply:
I remember now, I wanted to use `kubectl apply` so that it'd work even if the namespace already existed, but must have accidentally used `create` here instead. I'll gop with your suggestion of `exit_on_error=False`